src/modules/out_card10.c

```diff
@@ -0,0 +1,92 @@
+// Matrix order and size
+#if !defined(MATRIX_ORDER_PLAIN) && !defined(MATRIX_ORDER_SNAKE)
+#define MATRIX_ORDER_PLAIN
+#endif
+
+#ifndef MATRIX_X
+#error Define MATRIX_X as the matrixes X size.
+#endif
+
+#ifndef MATRIX_Y
+#error Define MATRIX_Y as the matrixes Y size.
+#endif
+
+#define MATRIX_PIXELS (MATRIX_X * MATRIX_Y)
+
+#include "../ext/card10/epicardium.h"
+
+#include "FreeRTOS.h"
+#include <types.h>
+#include <timers.h>
+#include <matrix.h>
+#include "timers.h"
+#include <stdint.h>
+#include <string.h>
+#include <stdio.h>
+#include <colors.h>
+
+static int counter = 0;
+union disp_framebuffer fb;
+
+int init(void) {
+	return epic_disp_open();
+}
+
+int deinit(void) {
+	return epic_disp_close();
+}
+
+int getx(void) {
+	return MATRIX_X;
+}
+int gety(void) {
+	return MATRIX_Y;
+}
+
+int ppos(int x, int y) {
+#ifdef MATRIX_ORDER_PLAIN
+	return (x + (y * MATRIX_X));
+#elif defined(MATRIX_ORDER_SNAKE)
+	// Order is like this
+	// 0 1 2
+	// 5 4 3
+	// 6 7 8
+	return (((y % 2) == 0 ? x : (MATRIX_X - 1) - x) + MATRIX_X*y);
+#endif
+}
+
+RGB get(int _modno, int x, int y) {
+	//size_t px = ppos(x, y);
+	// TODO
+	return RGB(0,0,0);
```

Comment:
shouldn't be hard to implement, probably'll do it myself.

Reply:
:+1: